src/destructors.md

```diff
@@ -104,6 +104,26 @@ dropped last when evaluating the function. Each actual function parameter is
 dropped after any bindings introduced in that parameter's pattern.
 
 ```rust
+// entire function scope begins
```

Comment:
First letter should be capitalized, for correctness and consistency with other Rust documentation (including this file). Ditto for all the other comments.